actors/evm/tests/call.rs

```diff
@@ -432,21 +428,7 @@ fn test_callactor_revert() {
 }
 
 // Much taken from tests/env.rs
-abigen!(CallActorPrecompile, "./tests/contracts/CallActorPrecompile.abi");
```

Comment:
should we just delete all the .abi files now?

Reply:
Yes.